test/metabase/query_processor/card_test.clj

```diff
@@ -151,12 +151,12 @@
 (deftest ^:parallel validate-card-parameters-test-4
   (mt/with-temp [:model/Card {card-id :id} {:dataset_query (field-filter-query)}]
     (testing "Happy path -- API request should succeed if parameter is valid"
-      (is (= [1000]
+      (is (= [6]
              (mt/first-row (mt/user-http-request :rasta :post (format "card/%d/query" card-id)
                                                  {:parameters [{:id    "_DATE_"
                                                                 :name  "date"
                                                                 :type  :date/single
-                                                                :value "2016-01-01"}]})))))))
+                                                                :value "2014-05-07"}]})))))))
```

Comment:
This test was wrong to begin with. It expected 1000 rows, but no row has `date = '2016-01-01'`. The change to `src/metabase/query_processor/card.clj` fixed the query and made test fail.